pkg/processor/runtime/python/py/_nuclio_wrapper.py

```diff
@@ -323,7 +323,7 @@ def run_wrapper():
                                    args.worker_id,
                                    args.trigger_name)
 
-    except Exception as exc:
+    except (Exception, GeneratorExit) as exc:
```

Comment:
Please add another except block & add `BaseException` to simply catch all exceptions that do not inherit from `Exception` (`GeneratorExit` is just one of them, but you have `SystemExit` and `KeyboardInterrupt` as well)

It is important to add that block AFTER the `except Exception as exc`

Reply:
If we want to catch `SystemExit` and `KeyboardInterrupt` as well, why not just catch `BaseException`? Why use a separate block?